source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -53,6 +189,10 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::filter>,
   // Return true if it is OK to forward this request without JWT.
   bool okToBypass() const;
 
+  // Returns the status true and first JwtVerifierPtr matching path, else return false and
+  // nullptr.
+  const std::pair<bool, const JwtMatcherConstPtr> tryMatch(const Http::HeaderString& path) const;
```

Comment:
from the comment, you don't need a pair, just return a JwtMatcherConstPtr is enough